snapdenv/useragent_test.go

```diff
@@ -64,6 +73,20 @@ func (s *UASuite) TestUserAgent(c *C) {
 	c.Check(strings.Contains(ua, "devmode"), Equals, true)
 }
 
+func (s *UASuite) TestUserAgentWSL(c *C) {
```

Comment:
given that this is the only test needing to mock OnWSL values, it could be done using testutil.Backup directly:

```
restore := testutil.Backup(&release.OnWSL)
defer restore()
release.OnWSL = false
...
release.OnWSL = true
...
```

Reply:
Alrighty. Fixed.